src/client/reducers/active-rooms.js

```diff
@@ -0,0 +1,12 @@
+import * as types from '../actions/types';
+
+export default (state = [], action) => {
+    switch(action.type) {
+        case types.UPDATE_ROOM_LIST:{
+            return action.payload.slice();
```

Comment:
it puts the payload into a new array, to do with immutable data error we were getting